src/load_paths.pl

```diff
@@ -48,9 +50,9 @@
     asserta(user:file_search_path(config, Config)).
 
 :- add_config_path.
+:- use_module(config(terminus_config)).
```

Comment:
This was done for several reasons:

- The linter does not pick up modules that are only imported inside a specific predicate
- I do think it sets a bad precedent if we import modules inside predicates instead of at the top of the file

As for side effects, I can't think of any but its good to be wary of.